plugins/win-dshow/CMakeLists.txt

```diff
@@ -1,150 +1,175 @@
 if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/libdshowcapture/dshowcapture.hpp")
-	message(STATUS "libdshowcapture submodule not found!  Please fetch submodules.  win-dshow plugin disabled.")
-	return()
+  message(
+    FATAL_ERROR
+      "OBS:   libdshowcapture submodule not found!  Please fetch submodules. win-dshow plugin disabled."
```

Comment:
Nitpick:  double spaces after one sentence, but one space after the next.

Reply:
Fixed.